main/src/test/java/org/mobilitydata/gtfsvalidator/validator/StopTimeStopLocationTypeValidatorTest.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2021 MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.validator;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.notice.ValidationNotice;
+import org.mobilitydata.gtfsvalidator.table.GtfsLocationType;
+import org.mobilitydata.gtfsvalidator.table.GtfsStop;
+import org.mobilitydata.gtfsvalidator.table.GtfsStopTableContainer;
+import org.mobilitydata.gtfsvalidator.table.GtfsStopTime;
+import org.mobilitydata.gtfsvalidator.table.GtfsStopTimeTableContainer;
+import org.mobilitydata.gtfsvalidator.validator.StopTimeStopLocationTypeValidator.WrongStopTimeStopLocationTypeNotice;
+
+@RunWith(JUnit4.class)
+public class StopTimeStopLocationTypeValidatorTest {
+
+  private static GtfsStopTime createStopTime(
+      long csvRowNumber, String tripId, String stopId, int stopSequence) {
+    return new GtfsStopTime.Builder()
+        .setCsvRowNumber(csvRowNumber)
+        .setTripId(tripId)
+        .setStopId(stopId)
+        .setStopSequence(stopSequence)
+        .build();
+  }
+
+  private static GtfsStop createStop(long csvRowNumber, String stopId, Integer locationTypeValue) {
```

Comment:
Should we use GtfsLocationtype in the signature instead since it is an Enum with limited Integer possibilities? No strong feeling about this, I just think it would be closer to the real world behaviour where receiving a value outside the Enum possibilities would be a problem.

Reply:
Good catch - yes!